### PR TITLE
Make submit and back button not appear if gift card is expired

### PR DESCRIPTION
**Pull request by Cloud11PL**
I want to merge this change because I want to make submit and back button appear in gift card update page only when gift card is not expired.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4473 In gift card update view - hide savebar buttons (besides delete) if gift card is expired](https://app.clickup.com/t/2549495/SALEOR-4473)

File: src/giftCards/GiftCardUpdate/messages.ts [hunk 1]
```diff
@@ -12,6 +12,21 @@ export const giftCardUpdateDetailsCardMessages = defineMessages({
...
+  back: {
+    defaultMessage: "Back",
+    description: "back button"
+  },
+  save: {
+    defaultMessage: "Save",
+    description: "save button"
+  }
+});
+
```
Comment (orzechdev) [inline src/giftCards/GiftCardUpdate/messages.ts:29]: Let's use `import { commonMessages } from "@saleor/intl"` instead of defining new messages

**Review by orzechdev — CHANGES_REQUESTED** [review]

File: src/giftCards/GiftCardUpdate/messages.ts (hunk 1 above)
Comment (Cloud11PL) [reply src/giftCards/GiftCardUpdate/messages.ts]: I didn't know such thing existed, thanks 😛

**Review by orzechdev — APPROVED** [review]

**Review by dominik-zeglen — APPROVED** [review]

**Comment by Cloud11PL** [comment]
Hi @dominik-zeglen can I ask you to test it? :)

**Pull request closed without merging**